CHANGES.md

```diff
@@ -15,7 +15,32 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
-[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
+## Changes
+
+- GeoServer: upgrade to 2.25.2 to fix vulnerabilities
+
+  See
+  https://nsfocusglobal.com/remote-code-execution-vulnerability-between-geoserver-and-geotools-cve-2024-36401-cve-2024-36404-notification/,
+  https://github.com/geoserver/geoserver/security/advisories/GHSA-6jj6-gm7p-fcvv,
+  https://github.com/geotools/geotools/security/advisories/GHSA-w3pj-wh35-fq8w
```

Comment:
Please make them as list items, easier to read after when the text is wrapped in HTML.

Reply:
Done 03f0577483c7b6ab6fa126964f1921085242de41